### PR TITLE
Lpd 25789 install safe packages in base dockerfile

### PR DESCRIPTION
**Pull request by Wykenz**

### CODE REVIEW

File: templates/base/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
-FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
+FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
```
Comment (natocesarrego) [inline templates/base/Dockerfile]: @Wykenz, do we really need to include the `adduser` command here? Isn't it installed in Ubuntu Noble by default?

File: templates/base/Dockerfile [hunk 2]
```diff
@@ -3,10 +3,28 @@ FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash build-essential ca-certificates curl jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip && \
```
Comment (natocesarrego) [inline templates/base/Dockerfile:6]: @Wykenz, can you please remove the additional packages as you did in https://github.com/natocesarrego/liferay-docker/pull/31/commits/40db715f245cc6a32948f86a9676c2df711edc8a?

File: templates/base/Dockerfile [hunk 3]
```diff
@@ -3,10 +3,28 @@ FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
...
 	apt-get upgrade --yes && \
 	apt-get clean
 
+# Fix https://ubuntu.com/security/CVE-2024-2398
+#
+# START
+RUN VERSION=8.7.1 && \
```
Comment (natocesarrego) [inline templates/base/Dockerfile:13]: @Wykenz, can you please move the curl installation to be bellow the inclusion of the `liferay` user?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: templates/base/Dockerfile (hunk 1 above)
Comment (Wykenz) [reply templates/base/Dockerfile]: It is not a default package somehow. I added it because it gave me a "command not found" error

**Pull request closed without merging**